pillar/edx/xqwatcher.sls

```diff
@@ -98,6 +98,44 @@ edx:
       {% endfor %}
       {% endif %}
       {% endfor %}
+      {% set 686_xqueue_creds = salt.vault.read('secret-residential/global/course-686-xqueue-credentials') %}
+      - COURSE: mit-686x
```

Comment:
Why did we move away from adding `purpose` and `queue_name` to course similar to `6.001`?

Reply:
This is because the 686 course is running on edX, not on residential so there is no different queue for different purposes.